rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_reader :person, :book
```

Comment:
- [ ] Setters for `@book` and `@person` instance variables should be present since a rental belongs to a book and also belongs to a person.